README.md

```diff
@@ -1,36 +1,46 @@
+[![early-release]][tracker-classification] [![Build][github-image]][github] [![Release][release-image]][releases] [![License][license-image]][license]
 
 ## Overview
 
 Add analytics to your Java software with the **[Snowplow][snowplow]** event tracker for **[Java][java]**. See also: **[Snowplow Android Tracker][snowplow-android-tracker]**.
 
 With this tracker you can collect event data from your Java-based desktop and server apps, servlets and games. Supports JDK7+.
 
-## Quickstart
+## Find out more
+
+| Snowplow Docs                   | Contributing                      |
+|---------------------------------|-----------------------------------|
+| ![i1][techdocs-image]           |  ![i4][contributing-image]         |
+| **[Snowplow Docs][techdocs]**  | **[Contributing](CONTRIBUTING.md)**  |
+
+## Maintainer Quickstart
+
+Feedback and contributions are very welcome. If you have identified a bug, please log an issue on this repo. For all other feedback, discussion or questions please open a thread on our [Discourse forum][forums]. Feel free to make Pull Requests for new features, if you can code them yourself!
 
-Assuming git, **[Vagrant][vagrant-install]** and **[VirtualBox][virtualbox-install]** installed:
+Clone this repo and navigate into the cloned folder. To run the tests locally, you will need Docker or Java installed. Using either method, the build will fail if there are failing tests.  
+
+To run the tests using Docker, run:
 
 ```bash
- host$ git clone https://github.com/snowplow/snowplow-java-tracker.git
- host$ cd snowplow-java-tracker
- host$ vagrant up && vagrant ssh
-guest$ cd /vagrant
-guest$ ./gradlew clean build
-guest$ ./gradlew test
-guest$ ./gradlew publishToMavenLocal
-guest$ cd /examples/simple-console
-guest$ ./gradlew jar
-guest$ java -jar ./build/libs/simple-console-all-0.0.1.jar "http://<your-collector-domain>"
+$ docker build -t snowplow-java-tracker
 ```
 
-## Find out more
+To run the tests using your installed JDK, run:
 
-| Technical Docs                  | Setup Guide               | Roadmap                 | Contributing                      |
-|---------------------------------|---------------------------|-------------------------|-----------------------------------|
-| ![i1][techdocs-image]           | ![i2][setup-image]        | ![i3][roadmap-image]    | ![i4][contributing-image]         |
-| **[Technical Docs][techdocs]**  | **[Setup Guide][setup]**  | **[Roadmap][roadmap]**  | **[Contributing](Contributing.md)**  |
+```bash
+$ ./gradlew build
+```
+
+We have also included a simple demo, found in the `examples/simple-console` folder. You will need a JDK installed to run it. When run, it sends several events to your event collector.
+
+```bash
+$ ./gradlew publishToMavenLocal
+$ ./examples/simple-console/gradlew jar
+$ java -jar ./examples/simple-console/build/libs/simple-console-all-0.0.1.jar "http://<your-collector-domain>"
```

Comment:
I would recheck these two steps with a fresh repo check-out.

`./examples/simple-console/gradlew jar` executes the gradlew script of simple-console in the current directory, creating the build folder in the current directory.
So, when you execute `java -jar ./examples/simple-console/build/...` the build folder is not present.
Please, correct me if I'm wrong.

Also, we updated `gradlew` script of simple-console with gradle 7.2 but the tracker library is still with the gradle 6.5 version.
I would update it to 7.2 too, unless it breaks any compatibility with JDK8 or other dependencies.
wdyt?

Reply:
You're right, it doesn't work. I think the problem is that the `gradle-wrapper.jar` of simple-console is not committed. The .gitignore includes `*.jar` - but strangely, the main project `gradle-wrapper.jar` is present. If the .gitignore is overriden for a certain file, would that be written somewhere?

Reply:
Interesting, I haven't noticed this.
I think the jar is there because it was already committed. If you remove it and commit it. It wouldn't be able to add it again on git because gitignore would stop you.
I think we should remove `*.jar` from `gitignore` so it doesn't remove the gradle-wrapper files.
The important thing is that the `build` folder is excluded as that is the destination of all the jars created at compile-time.

Anyway, the other problem is that the `gradlew` script should be launched from `simple-console` folder, otherwise it doesn't create the app jar in the right path (e.g: ./examples/simple-console/build/).